packages/usa-input/src/usa-input.twig

```diff
@@ -1,4 +1,4 @@
-<label class="usa-label" for="input-type-text">Text input label</label>
+<label class="usa-label test" for="input-type-text">Text input label</label>
```

Comment:
What does `test` do?

Reply:
This has been removed and will be reflected in the next push I make tonight, thanks!

Reply:
Thanks!